server/events/vcs/bitbucketserver/client.go

```diff
@@ -137,6 +137,8 @@ func (b *Client) CreateComment(repo models.Repo, pullNum int, comment string, co
 	sepEnd := "\n```\n**Warning**: Output length greater than max comment size. Continued in next comment."
 	sepStart := "Continued from previous comment.\n```diff\n"
 	comments := common.SplitComment(comment, maxCommentLength, sepEnd, sepStart)
+	common.ReverseComments(comments)
```

Comment:
As of 1.21 (which atlantis uses), go has a standard library module `slices` for generic slice operations, including `Reverse()`. I'd recommend this be used and the test removed.
```suggestion
	slices.Reverse(comments)
```